### ISSUE TITLE
Trouble getting tox to build anything from the matrix

### ISSUE THREAD

**Issue by stefanfoulis** [ymyzk/tox-gh-actions#1]
Probably I just configured something wrong, but I'm having trouble getting `tox` to run any tests.

https://github.com/stefanfoulis/django-sendsms-gh-ci-experiment/pull/1/checks?check_run_id=222843429

action: https://github.com/stefanfoulis/django-sendsms-gh-ci-experiment/blob/ci/gh-actions-experiment/.github/workflows/ci.yml
tox.ini: https://github.com/stefanfoulis/django-sendsms-gh-ci-experiment/blob/ci/gh-actions-experiment/tox.ini

In the same repo I also have a setup with travis using tox-travis which works.

Any help would be greatly appreciated.

**Comment by ymyzk** [comment]
@stefanfoulis thanks for your feedback!

I think tox-gh-actions cannot handle this line `envlist = py27-dj{111}, py{35,36,37}-dj{111,20}, flake8, black` yet. I'd like to fix the issue.

A workaround I can think of is updating `tox.ini` like this:
```
[gh-actions]
python =
  2.7: py27-dj111
  3.5: py35-dj111, py35-dj20
  3.6: py36-dj111, py36-dj20
  3.7: py37-dj111, py37-dj20, black
```

Could you give it a try? Thanks again.

**Comment by stefanfoulis** [comment]
@ymyzk thanks for the quick response. Yes, if I change the config as you suggest it runs the tests👍

**Comment by ymyzk** [comment]
Awesome! Thank you for trying the library.

**Comment by ymyzk** [comment]
@stefanfoulis I just released tox-gh-actions v0.2.0 with an improved algorithm. The following `tox.ini` should work fine now.

```
[gh-actions]
python =
  2.7: py27
  3.5: py35
  3.6: py36
  3.7: py37, black
```

**Comment by stefanfoulis** [comment]
awesome! I'll try this out asap.
